code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -70,17 +70,49 @@
 	mulebot_count += 1
 	set_id(suffix || id || "#[mulebot_count]")
 	suffix = null
+	var/datum/component/riding/D = LoadComponent(/datum/component/riding)
```

Comment:
this was loaded in simple/bot's buckle_mob, so no change in adding it in Initialize aside from the relevant fixes.